tree/ntuple/v7/src/libdaos_mock/libdaos_mock.cxx

```diff
@@ -169,18 +158,18 @@ class RDaosFakeContainer {
 class RDaosFakePool {
 private:
    static std::mutex fMutexPools;
-   static std::unordered_map<Uuid_t, std::unique_ptr<RDaosFakePool>> fPools;
+   static std::unordered_map<label_t, std::unique_ptr<RDaosFakePool>> fPools;
 
    std::mutex fMutexContainers;
-   std::unordered_map<Uuid_t, std::unique_ptr<RDaosFakeContainer>> fContainers;
+   std::unordered_map<label_t, std::unique_ptr<RDaosFakeContainer>> fContainers;
 
 public:
    /// \brief Get a pointer to a RDaosFakePool object associated to the given UUID.
    /// Non-existent pools shall be created on-demand.
-   static RDaosFakePool *GetPool(const Uuid_t uuid)
+   static RDaosFakePool *GetPool(const label_t label)
```

Comment:
We should probably take a reference type instead (same for other uses, e.g. in `CreateContainer()`).
```suggestion
   static RDaosFakePool *GetPool(const label_t &label)
```